setup.py

```diff
@@ -22,47 +22,53 @@ def read(*rnames):
 
 long_description = (read('README.rst') + '\n\n' + read('CHANGES.rst'))
 
-setup(name='zope.ptresource',
-      version='4.1.0.dev0',
-      url='http://pypi.python.org/pypi/zope.ptresource/',
-      author='Zope Foundation and Contributors',
-      author_email='zope-dev@zope.org',
-      classifiers = [
-          'Environment :: Web Environment',
-          'Intended Audience :: Developers',
-          'License :: OSI Approved :: Zope Public License',
-          'Programming Language :: Python',
-          'Programming Language :: Python :: 2',
-          'Programming Language :: Python :: 2.7',
-          'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.3',
-          'Programming Language :: Python :: 3.4',
-          'Programming Language :: Python :: 3.5',
-          'Programming Language :: Python :: Implementation :: CPython',
-          'Programming Language :: Python :: Implementation :: PyPy',
-          'Operating System :: OS Independent',
-          'Topic :: Internet :: WWW/HTTP',
-          'Framework :: Zope3',
-        ],
-      description='Page template resource plugin for zope.browserresource',
-      long_description=long_description,
-      license='ZPL 2.1',
-      packages=find_packages('src'),
-      package_dir={'': 'src'},
-      namespace_packages=['zope'],
-      install_requires=[
+TESTS_REQUIRE = [
+    'zope.testing',
+    'zope.testrunner',
+]
+
+setup(
+    name='zope.ptresource',
+    version='4.1.0.dev0',
+    url='http://github.com/zopefoundation/zope.ptresource/',
```

Comment:
I would prefer https://

Also, the trailing slash looks weird to me, although I suppose it works.

Reply:
I've pretty much just been doing `s/pypi.python.org\/pypi/github.com\/zopefoundation`. http redirects to https, so that's something I guess 🤷‍♂️